sympy/parsing/autolev/__init__.py

```diff
@@ -47,10 +47,9 @@ def parse_autolev(autolev_code, include_numeric=False, numpy_import=False):
     ... "CODE DYNAMICS() some_filename.c")
     >>> my_al_text = '\\n'.join(my_al_text)
     >>> from sympy.parsing.autolev import parse_autolev
-    >>> print(parse_autolev(my_al_text, include_numeric=True))
+    >>> print(parse_autolev(my_al_text, include_numeric=True, numpy_import=True))
```

Comment:
I thought the `include_numeric` flag added the numpy related code. I don't think we need an additional flag.

Reply:
So currently does the issue still exist? Isn't the numpy related code currently added regardless of any flag?

Reply:
Some imports are not needed if include_numeric is false.

Reply:
If `include_numeric` is False, there should be no numpy or pydy imports in the generated code. The only imports should be from sympy.